lib/appSession.js

```diff
@@ -9,7 +9,7 @@ const onHeaders = require('on-headers');
 const cookie = require('cookie');
 const COOKIES = require('./cookies');
 const { encryption: deriveKey } = require('./hkdf');
-const debug = require('./debug');
+const debug = require('./debug')('appSession');
```

Comment:
I removed the levels because they didn't work like logging levels (`warn:*` wouldn't include `info:*` etc).

But I liked how express split the logs up by filename (https://github.com/expressjs/express/blob/master/lib/application.js#L21)